Tools-Override/tests.targets

```diff
@@ -201,8 +199,10 @@
 
     <!-- For .NET Native compilation, we first need to generate a native executable if possible. -->
     <ItemGroup Condition="'$(UseDotNetNativeToolchain)' == 'true' AND '$(Performance)'!='true'" >
```

Comment:
Where does UseDotNetNativeToolchain get set to true? Should it be defaulted base on TestILCFolder?